vector/src/main/java/im/vector/app/features/home/room/list/RoomListViewModel.kt

```diff
@@ -338,4 +320,16 @@ class RoomListViewModel @AssistedInject constructor(
             _viewEvents.post(value)
         }
     }
+
+    private fun handleDeleteLocalRooms() {
+        val localRoomIds = session.roomService()
+                .getRoomSummaries(roomSummaryQueryParams { roomId = QueryStringValue.Contains(RoomLocalEcho.PREFIX) })
+                .map { it.roomId }
+
+        viewModelScope.launch {
+            localRoomIds.forEach {
+                session.roomService().deleteLocalRoom(it)
```

Comment:
Not for this PR, but having a method `RoomService.deleteAllLocalRooms()` would be handy.